src/GovernorCountingFractional.sol

```diff
@@ -91,14 +102,19 @@ abstract contract GovernorCountingFractional is Governor {
     /**
      * @notice See {Governor-_countVote}.
      *
-     * If the `voteData` bytes parameter is empty, then this module behaves
-     * identically to GovernorBravo. That is, it assigns the full weight of the delegate to the `support` parameter,
-     * which follows the `VoteType` enum from Governor Bravo.
+     * @dev If the `voteData` bytes parameter is empty, then this module behaves
+     * identically to GovernorBravo. That is, it assigns the full weight of the
+     * delegate to the `support` parameter, which follows the `VoteType` enum
+     * from Governor Bravo.
      *
-     * If the `voteData` bytes parameter is not zero, then it _must_ be three packed uint128s, totaling 48 bytes,
-     * representing the weight the delegate assigns to Against, For, and Abstain respectively, i.e.
-     * encodePacked(againstVotes, forVotes, abstainVotes). The sum total of the three decoded vote weights _must_ be
-     * less than or equal to the delegate's total weight as check-pointed for the proposal being voted on.
+     * If the `voteData` bytes parameter is not zero, then it _must_ be three
+     * packed uint128s, totaling 48 bytes, representing the weight the delegate
+     * assigns to Against, For, and Abstain respectively, i.e.
+     * encodePacked(againstVotes, forVotes, abstainVotes). The sum total of
+     * the three decoded vote weights _must_ be less than or equal to the
+     * delegate's total weight as checkpointed by the proposal being voted on.
```

Comment:
Feel free to tweak, but I think it conveys the idea
```suggestion
     * delegate's total uncast weight. i.e. their checkpointed total weight minus votes already cast.
```

Reply:
Good clarification, thanks. See d301ac2